packages/core/contracts/oracle/implementation/OptimisticOracle.sol

```diff
@@ -163,8 +164,11 @@ contract OptimisticOracle is OptimisticOracleInterface, Testable, Lockable {
         require(_getIdentifierWhitelist().isIdentifierSupported(identifier), "Unsupported identifier");
         require(_getCollateralWhitelist().isOnWhitelist(address(currency)), "Unsupported currency");
         require(timestamp <= getCurrentTime(), "Timestamp in future");
+
+        // TODO: to make this cheaper, we should just compute the max length of the stamping and add it to the
+        // ancillary data length.
```

Comment:
Can you explain this change? Are you basically passing in the longest possible ancillary data length and checking that its under the limit?

Reply:
Yep, exactly. In the future, I'd like to just get a constant value for the max added length and add it to avoid the costs of constructing the ancillary data during this call.